src/radio-group/interfaces.ts

```diff
@@ -40,6 +40,11 @@ export interface RadioGroupProps extends BaseComponentProps, FormFieldControlPro
    */
   ariaRequired?: boolean;
 
+  /**
+   * Adds `aria-controls` attribute to the radio group. Use this property if the radio group controls the visibility of other elements on the page.
```

Comment:
How about this

```suggestion
   * Adds `aria-controls` attribute to the radio group. If radio group selection controls presence of another form field, use this property to provide an id the controlled field
```

Reply:
Still would like to hear what you think about this suggestion

Reply:
I'm generally open to the change, but not so sure about the suggestion.

I'm not sure if this part is syntactically correct: `to provide an id the controlled field`.

How about this instead?

```js
   /**
    * Adds `aria-controls` attribute to the radio group. 
    * If the radio group controls the visibility of any secondary content (like another FormField), provide an ID that refers to the secondary content.
    */

```

Reply:
Updated API doc following offline discussion